flyway/gxa/migrations/V9__gxa-create-exp-design.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE gxa_marker_gene
+(
+    experiment_accession VARCHAR(255) CHECK (experiment_accession LIKE 'E-%') NOT NULL,
+    assay VARCHAR(255) NOT NULL,
+    assay_id VARCHAR(255) NOT NULL,
+    gene_id VARCHAR(255) NOT NULL,
+    gene_name VARCHAR(255) NOT NULL,
+    marker_gene_rank INTEGER,
+    expression_level FLOAT NOT NULL,
+    expression_unit VARCHAR(8) NOT NULL,
+    number_assays INTEGER NOT NULL,
+    specificity_score FLOAT,
+    CONSTRAINT gxa_marker_gene_pkey PRIMARY KEY (experiment_accession, assay, gene_id)
```

Comment:
`PRIMARY KEY (experiment_accession, assay, gene_id)` is fine if this combo uniquely identifies a row. 

As here we will have fpkms and  tpms, should we add also `expression_unit`?